code/modules/reagents/newchem/drinks.dm

```diff
@@ -62,3 +62,117 @@
 	result = "jackrose"
 	required_reagents = list("applejack" = 4, "lemonjuice" = 1)
 	result_amount = 5
+
+
+// ROBOT ALCOHOL PAST THIS POINT
+// WOOO!
+
+
+/datum/reagent/alkasine //Gloriously copied over from ethanol
+	name = "Alkasine"
+	id = "alkasine"
+	description = "A runny liquid with conductive capacities. Its effects on synthetics are similar to those of alcohol on organics."
+	reagent_state = LIQUID
+	color = "#1BB1FF"
+	process_flags = SYNTHETIC
+	metabolization_rate = 0.4
+	can_grow_in_plants = 0	//Alcoholic drinks won't be grown in plants (would "water down" random seed chems too much)
+	var/datum/martial_art/drunk_brawling/F = new
+	var/dizzy_adj = 3
+	var/slurr_adj = 3
+	var/confused_adj = 2
+	var/slur_start = 65			//amount absorbed after which mob starts slurring
+	var/brawl_start = 75
+	var/confused_start = 130	//amount absorbed after which mob starts confusing directions and sparking
+	var/collapse_start = 200	//amount absorbed after wich mob starts collapsing
+	var/braindamage_start = 300 //amount absorbed after which mob starts taking small amount of brain damage
+
+
+/datum/chemical_reaction/alkasine
+	name = "Alkasine"
+	id = "alkasine"
+	result = "alkasine"
+	required_reagents = list("lube" = 1, "plasma" = 1, "fuel" = 1)
+	result_amount = 3
+	mix_message = "The chemicals mix to create a shiny, blue substance."
+
+/datum/reagent/alkasine/on_mob_life(var/mob/living/M as mob, var/alien)
+	if(!src.data) data = 1
+	src.data++
+
+	var/d = data
+
+	// make all the alkasine children work together
+	for(var/datum/reagent/alkasine/A in holder.reagent_list)
+		if(isnum(A.data)) d += A.data
+
+	M.dizziness += dizzy_adj.
+	if(d >= slur_start)
+		if (!M:slurring) M:slurring = 1
```

Comment:
COLON OPERATORS?!

If you think I"m mad, just wait until @TheDZD wake up, you won't live ot see the rest of the day.